src/main/resources/application.properties

```diff
@@ -122,10 +122,10 @@ server.error.include-stacktrace=never
 # PDF generator configuration
 nae.pdf.resources.outputFolder=storage
 nae.pdf.resources.outputDefaultName=generated_pdf.pdf
-nae.pdf.resources.templateResource=file:src/main/resources/pdfGenerator/template_pdf.pdf
-nae.pdf.resources.font-title-resource=file:src/main/resources/pdfGenerator/fonts/Roboto-Bold.ttf
-nae.pdf.resources.font-label-resource=file:src/main/resources/pdfGenerator/fonts/Roboto-Bold.ttf
-nae.pdf.resources.font-value-resource=file:src/main/resources/pdfGenerator/fonts/Roboto-Bold.ttf
+nae.pdf.resources.templateResource=classpath:pdfGenerator/template.pdf
```

Comment:
What was wrong with loading font resources from an external file? Why change it to classpath?

Reply:
There was an issue with template files, that in some cases the system cannot found the file based on "file:" resource, so I decided to make a classpath resource, that works in every cases.